fields/types/cloudinaryimages/CloudinaryImagesField.js

```diff
@@ -16,8 +16,8 @@ import Thumbnail from './CloudinaryImagesThumbnail';
 import HiddenFileInput from '../../components/HiddenFileInput';
 import FileChangeMessage from '../../components/FileChangeMessage';
 
-const SUPPORTED_TYPES = ['image/*', 'application/pdf', 'application/postscript'];
-const SUPPORTED_REGEX = new RegExp(/^image\/|application\/pdf|application\/postscript/g);
+const SUPPORTED_TYPES = ['video/*', 'image/*', 'application/pdf', 'application/postscript'];
```

Comment:
@JedWatson I hadn't finished working out video support on the single CloudinaryImage yet. Will work on it soon.